package.json

```diff
@@ -13,13 +13,15 @@
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.24.2",
     "hint": "^6.1.4",
+    "html-loader": "^2.1.2",
```

Comment:
- [ ] Script like `start` is missing in your package.json. Kindly try to add the correct script to start the project locally.

Webpack configured script
![image](https://user-images.githubusercontent.com/53019916/131649690-2d6e8991-558b-43db-9f1c-c4eb9594b7d3.png)

Reply:
Successfully Added the start script in the **package.json** file